src/main/webapp/app/exam/participate/timer/exam-timer.scss

```
@@ -9,11 +9,10 @@
 }
 .remaining-time {
     text-align: center;
-    min-width: 90px;
-
-    &.endViewTime {
-        min-width: 120px;
-    }
+    min-width: 80px;
+}
+.end-view {
```

Codacy wants you to insert a newline here:

```suggestion
}

.end-view {
```